backend/logistica_servicio/gestor_entregas/tests/test_repositorio_entrega_camion.py

```
@@ -0,0 +1,92 @@
+import unittest
+from infraestructura.repositorio_entrega_camion import RepositorioEntregasProgramadas, RepositorioEntregaProgramadasDetalle, Session
```

Ruff (0.8.2)

`infraestructura.repositorio_entrega_camion.RepositorioEntregaProgramadasDetalle` imported but unused

Remove unused import: `infraestructura.repositorio_entrega_camion.RepositorioEntregaProgramadasDetalle`

(F401)